From cdb098617af97c6af76e9048af62e8e5e778b5b6 Mon Sep 17 00:00:00 2001
From: Jo Shields <jo.shields@xamarin.com>
Date: Tue, 11 Aug 2015 19:41:23 +0100
Subject: [PATCH] Un-revoke fix in BNC#144655 - mono-find-provides should ONLY
 work on GAC libraries, and any packages which break on this behaviour should
 be fixed - rather than generating utterly broken Provides.

(cherry picked from commit 1f4133a06f252ec1b78637dd91e49f50836cf570)
---
 scripts/mono-find-provides.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/mono-find-provides.in b/scripts/mono-find-provides.in
index 155690d..5e440ab 100644
--- a/scripts/mono-find-provides.in
+++ b/scripts/mono-find-provides.in
@@ -14,7 +14,7 @@ monolist=($(printf "%s\n" "${filelist[@]}" | egrep "\\.(exe|dll)\$"))
 
 # Only include files with /gac/ in path
 #  (Allows packages to contain private assemblies that don't conflict with other packages)
-#monolist=($(printf "%s\n" "${monolist[@]}" | egrep "/gac/"))
+monolist=($(printf "%s\n" "${monolist[@]}" | egrep "/gac/"))
 # Disabled... see ChangeLog
 
 # Set the prefix, unless it is overriden (used when building mono rpms)
-- 
1.9.1

